routes/restaurants.js

```diff
@@ -2,7 +2,21 @@ const Restaurant = require('../models/restaurant.js').model
 
 exports.allRestaurants = async (req, res, next) => {
   try {
-    const docs = await Restaurant.find({})
+    const { price_range, genres } = req.query
+    const query = {}
+    const limit = req.query.limit ? Number(req.query.limit) : 10
+
+    if (price_range != null) {
+      query.price_range = price_range
```

Comment:
According to the doc, https://github.com/GLO3102/UFood/wiki/API:-Restaurants price_range should be a comma-delimited array too